src/ui/IconDropdownNav/index.js

```diff
@@ -69,7 +82,7 @@ export function IconDropdownNavItem ({ href, onClick, title, ...props }) {
 
       {
         onClick &&
-          <button onClick={onClick}>
+          <button onClick={handleOnClick}>
```

Comment:
JSX props should not use functions - `react/jsx-no-bind`

Får du sett på denne @ksundmyhr ?